charts/plgd-hub/README.md

```diff
@@ -245,10 +245,15 @@ global:
 | coapgateway.service.udp.type | string | `nil` | Service type |
 | coapgateway.taskQueue | object | `{"goPoolSize":1600,"maxIdleTime":"10m","size":"2097152"}` | For complete coap-gateway service configuration see [plgd/coap-gateway](https://github.com/plgd-dev/hub/tree/main/coap-gateway) |
 | coapgateway.tolerations | object | `{}` | Toleration definition |
+| extraAuthCaPool | object | `{"fileName":"ca.crt","mountPath":"/certs/extra","name":"extra-auth-ca-pool"}` | Configuration parameters for authorizationCaPool |
+| extraAuthCaPool.fileName | string | `"ca.crt"` | Name of file for custom auth ca pool |
+| extraAuthCaPool.mountPath | string | `"/certs/extra"` | Mount path for custom auth ca pool |
+| extraAuthCaPool.name | string | `"extra-auth-ca-pool"` | Name of secret for storing custom auth ca pool |
```

Comment:
Please keep naming consistent. The `authorization` below, `auth`here. Use full name, no worries that it's long.

Reply:
and make sure you write capital `CA`. That's how we use it everywhere in config, if the property doesn't start with ca. e.g.`useSystemCAPool`